DEVELOPER.md

```diff
@@ -0,0 +1,25 @@
+# Contributing
+
+## Linters
+
+This project uses a set of linters to ensure a good code quality.
+In order to make proper use of those linters inside an IDE, the following configurations are required.<br>
+Further information can also be found at the `golangci-lint` documentation [here](https://golangci-lint.run/usage/integrations/)
+
+### Visual Studio Code
+
+In Visual Studio Code the [Golang](https://marketplace.visualstudio.com/items?itemName=aldijav.golangwithdidi) extension is required.<br>
+Adding the following lines to the Golang extension configuration file will enable all linters used in this project.
+
+```
+"go.lintTool": "golangci-lint",
+"go.lintFlags": [
+  "--fast",
+  "--fix"
+]
+```
+
+### GoLand / IntelliJ
+
+In GoLand or IntelliJ, the plugin [Go Linter](https://plugins.jetbrains.com/plugin/12496-go-linter) will be required.
+Once installed, the configuration can be found in `Settings` >> `Tools` >> `Go Linter` <br>
```

Comment:
Maybe I would ass some more information about that, something like Anna showed here

![Screenshot from 2022-11-24 14-41-42](https://user-images.githubusercontent.com/93584209/203798684-6b42a3b5-67c5-4841-a151-2267c9ddf339.png)